### PR TITLE
Camera added (Sensors)

### PR DESCRIPTION
**Pull request by Terminateit**
Added Camera simulation and recording to the robot!
The video of how camera works:
[The video on youtube](https://youtu.be/JSsNXLpggLg)

To run the project type this in the terminal:
```bash
roslaunch atlas_launch control_camera.launch 
```

I added some Images to show the examples.

### CODE REVIEW

**Comment by dankirsdot** [comment]
Lol.

File: atlas_camera/CMakeLists.txt [hunk 1]
```diff
@@ -0,0 +1,202 @@
...
+## is used, also find other catkin packages
+find_package(catkin REQUIRED)
+
+## System dependencies are found with CMake's conventions
+# find_package(Boost REQUIRED COMPONENTS system)
+
+
+## Uncomment this if the package has a setup.py. This macro ensures
+## modules and global scripts declared therein get installed
+## See http://ros.org/doc/api/catkin/html/user_guide/setup_dot_py.html
```
Comment (l1va) [inline atlas_camera/CMakeLists.txt]: do you reall y need in all these comments?

File: atlas_camera/controller/.ipynb_checkpoints/main-checkpoint.ipynb [hunk 2]
```diff
@@ -0,0 +1,6 @@
+{
+ "cells": [],
```
Comment (l1va) [inline atlas_camera/controller/.ipynb_checkpoints/main-checkpoint.ipynb]: i think it is useless file, what is good to do with it?

File: atlas_camera/controller/main.py [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+if __name__ == '__main__':
+	time.sleep(5)
+	i = 0
+	image_topic = "/crobot/camera1/image_raw"
+	while (i<50):
+		rotate1('0',-math.pi,math.pi)
+		rotate1('1',-math.pi,0)
+		rotate1('2',-9,9)
+		rotate1('3',-math.pi/2,math.pi/2)
+		rospy.Subscriber(image_topic, Image, image_callback)
```
Comment (l1va) [inline atlas_camera/controller/main.py]: how many subscribers will you create?

File: atlas_camera/urdf/atlas.urdf.xacro [hunk 4]
```diff
@@ -0,0 +1,671 @@
+<?xml version="1.0" encoding="utf-8"?>	
+	<robot name="atlas" xmlns:xacro="http://ros.org/wiki/xacro">
```
Comment (l1va) [inline atlas_camera/urdf/atlas.urdf.xacro]: again urdf? why you need one more copy?

File: atlas_camera/controller/.ipynb_checkpoints/main-checkpoint.ipynb (hunk 2 above)
Comment (Terminateit) [reply atlas_camera/controller/.ipynb_checkpoints/main-checkpoint.ipynb]: Delete it?

File: atlas_camera/urdf/atlas.urdf.xacro (hunk 4 above)
Comment (Terminateit) [reply atlas_camera/urdf/atlas.urdf.xacro]: I thought It'd be cool to show the evolution of my project and not to mix old urdf for showing the model, for ex., and the new with the camera. Should I change it and use only the new one?

File: atlas_camera/controller/main.py (hunk 3 above)
Comment (Terminateit) [reply atlas_camera/controller/main.py]: I assume, 4

File: atlas_camera/urdf/atlas.urdf.xacro (hunk 4 above)
Comment (l1va) [reply atlas_camera/urdf/atlas.urdf.xacro]: in such case you can add addition urdf near the previous that contains only camera or addition values and not copy fully.

File: atlas_camera/controller/main.py (hunk 3 above)
Comment (l1va) [reply atlas_camera/controller/main.py]: no

File: atlas_camera/controller/.ipynb_checkpoints/main-checkpoint.ipynb (hunk 2 above)
Comment (l1va) [reply atlas_camera/controller/.ipynb_checkpoints/main-checkpoint.ipynb]: maybe better to add to gitignore? to not see the same in future? (if it is really not needed files)

File: atlas_camera/controller/main.py (hunk 3 above)
Comment (Terminateit) [reply atlas_camera/controller/main.py]: so many-many subscribers

File: atlas_camera/controller/.ipynb_checkpoints/main-checkpoint.ipynb (hunk 2 above)
Comment (Terminateit) [reply atlas_camera/controller/.ipynb_checkpoints/main-checkpoint.ipynb]: Solved it

File: atlas_camera/urdf/atlas.urdf.xacro (hunk 4 above)
Comment (Terminateit) [reply atlas_camera/urdf/atlas.urdf.xacro]: Solved it

File: atlas_camera/controller/main.py (hunk 3 above)
Comment (l1va) [reply atlas_camera/controller/main.py]: mm, do you think i'm just asking?

Comment (Terminateit) [reply atlas_camera/controller/main.py]: `Resolved`

**Pull request merged**